vue-components/package.json

```diff
@@ -10,8 +10,10 @@
     "build": "vue-cli-service build --target lib --formats commonjs --name wikit-vue-components src/main.ts --report --report-json",
     "test:unit": "vue-cli-service test:unit",
     "test:lint": "vue-cli-service lint --no-fix",
+    "test:lint:css": "stylelint '**/*.vue'",
```

Comment:
I'm not sure what exactly is going wrong, but it appears that this script does not get picked up by `npm-run-all test:*`. If I remove the second ":" it works fine. Odd.

Reply:
Yep, npm-run-all does not seem to support the double `:`
I would personally drop it in favor of `&&` but since this is not a concern of this PR, I'll just remove the second `:` for now.